src/core_plugins/kibana/public/management/sections/objects/_objects.js

```diff
@@ -187,6 +187,10 @@ uiModules.get('apps/management')
             obj.id = doc._id;
             return obj.applyESResp(doc).then(function () {
               return obj.save({ confirmOverwrite : true });
+            }).catch(err => {
```

Comment:
I might fix this after merging. I think I'm going to have merge conflicts to deal with anyway, but I'll get it updated.